llvm-spirv/lib/SPIRV/OCL20ToSPIRV.cpp

```diff
@@ -288,11 +288,31 @@ class OCL20ToSPIRV : public ModulePass, public InstVisitor<OCL20ToSPIRV> {
   Module *M;
   LLVMContext *Ctx;
   unsigned CLVer; /// OpenCL version as major*10+minor
+  unsigned CLLang; /// OpenCL language, see `spv::SourceLanguage`.
```

Comment:
Please note that this repo contains a copy of KrhonosGroup/SPIRV-LLVM-Translator which is updated from time to time and usually, by simple copy-pasting the whole directory. So, there is a huge chance that these changes will be overwritten occasionally

Please at least extract these changes to a separate commit, so it can be re-applied at each sync, or (which is much better) commit them directly to KhronosGroup/SPIRV-LLVM-Translator

Reply:
We will try to commit these directly to the Khronos repository, but will need the changes here until it is merged on the Khronos repository and pulled back down for the CUDA backend to work.

For now the changes to llvm-spirv are separated into https://github.com/intel/llvm/pull/1091/commits/09b3b2e401f03c08f3422f9700d8814da08fc43f